src/modules/m_nick.c

```diff
@@ -426,7 +426,9 @@ void nick_register (Nick *nptr, User *uptr, char *all)
         NoticeToUser(nptr, "A password has been generated and sent to your specified e-mail address (this mean that the password you've specified doesn't work).");
     } else {        
         user->authed = 1;
-        SendRaw("SVSMODE %s +r",nptr->nick);
+        nptr->svid = user->nick;
+        SendRaw("SVSMODE %s +r", nptr->nick);
```

Comment:
Can we do an SVS2MODE instead? That way, the user will see the mode +r being applied to themself.

Reply:
Done